products/visionos.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Vision OS
```

Comment:
```suggestion
title: Apple visionOS
```

We prefixed with 'apple' for macOS/iOS, so let's keep doing it. Also the name is wrongly spelled, see https://developer.apple.com/visionos/.